### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by mathieu-aubin**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ url: "http://developer.fedoraproject.org" # the base hostname & protocol for you
 twitter_username: jekyllrb
 github_username:  jekyll
 encoding: utf-8
+fedora_release: 34
```
Comment (jackorp) [inline _config.yml:12]: Why is it being declared here twice? Let's prevent duplication.

The latter declaration could work.

File: _includes/footer.html [hunk 2]
```diff
@@ -46,8 +46,8 @@ <h3 class="widget-title">Support</h3>
               <dd><a href="https://fedoraproject.org/wiki/Communicating_and_getting_help">Get Help</a></dd>
               <dd><a href="https://ask.fedoraproject.org/">Ask Fedora</a></dd>
               <dd><a href="https://fedoraforum.org/">Forums</a></dd>
-              <dd><a href="https://fedoraproject.org/wiki/Common_F25_bugs">Common Bugs</a></dd>
-              <dd><a href="https://docs.fedoraproject.org/en-US/Fedora/25/html/Installation_Guide">Installation Guide</a></dd>
+              <dd><a href="https://fedoraproject.org/wiki/Common_F{{ site.fedora_release }}_bugs">Common Bugs</a></dd>
+              <dd><a href="https://docs.fedoraproject.org/en-US/Fedora/{{ site.fedora_release }}/html/Installation_Guide">Installation Guide</a></dd>
```
Comment (jackorp) [inline _includes/footer.html:50]: This does not seem to be pointing to the right site (I am getting 404).

This one should be the right one.
```suggestion
              <dd><a href="https://docs.fedoraproject.org/en-US/fedora/f{ site.fedora_release }}/install-guide/">Installation Guide</a></dd>
```

**Review by jackorp — CHANGES_REQUESTED** [review]
Thanks for updating the links!

Please address the comments, then it can be merged.

File: _config.yml (hunk 1 above)
Comment (mathieu-aubin) [reply _config.yml:12]: it was already on the bottom but i don't have jekyll knowledge to reach that variable so i redeclared it as a 'site' global variable.

figured id leave the old config as it was so i don'T fuck up things

Comment (mathieu-aubin) [reply _config.yml:12]: it wouldnt generate a version without adding it in the general/default site variable

File: _config.yml [hunk 3]
```diff
@@ -104,4 +105,4 @@ defaults:
     scope:
       path: ""
     values:
-      fedora_release: "33"
+      fedora_release: "34"
```
Comment (mathieu-aubin) [inline _config.yml:108]: Maybe remove this as it's not being used i believe??

Comment (jackorp) [reply _config.yml:108]: This cannot be removed as some pages use this (for example copr-cli.md).

File: _includes/footer.html [hunk 4]
```diff
@@ -46,8 +46,8 @@ <h3 class="widget-title">Support</h3>
               <dd><a href="https://fedoraproject.org/wiki/Communicating_and_getting_help">Get Help</a></dd>
               <dd><a href="https://ask.fedoraproject.org/">Ask Fedora</a></dd>
               <dd><a href="https://fedoraforum.org/">Forums</a></dd>
-              <dd><a href="https://fedoraproject.org/wiki/Common_F25_bugs">Common Bugs</a></dd>
-              <dd><a href="https://docs.fedoraproject.org/en-US/Fedora/25/html/Installation_Guide">Installation Guide</a></dd>
+              <dd><a href="https://fedoraproject.org/wiki/Common_F{{ site.fedora_release }}_bugs">Common Bugs</a></dd>
```
Comment (jackorp) [inline _includes/footer.html:49]:
```suggestion
              <dd><a href="https://fedoraproject.org/wiki/Common_F{{ page.fedora_release }}_bugs">Common Bugs</a></dd>
```

File: _includes/footer.html [hunk 5]
```diff
@@ -46,8 +46,8 @@ <h3 class="widget-title">Support</h3>
...
+              <dd><a href="https://docs.fedoraproject.org/en-US/fedora/f{ site.fedora_release }}/install-guide/">Installation Guide</a></dd>
```
Comment (jackorp) [inline _includes/footer.html:50]:
```suggestion
              <dd><a href="https://docs.fedoraproject.org/en-US/fedora/f{{ page.fedora_release }}/install-guide/">Installation Guide</a></dd>
```

File: _config.yml (hunk 1 above)
Comment (jackorp) [inline _config.yml:12]: We can access the latter variable through `page` like this: `{{ page.fedora_release }}`, so this is not needed
```suggestion
```

**Review by jackorp — CHANGES_REQUESTED** [review]
I found the correct way of accessing the page variables, so we can have it defined in a single place. (I also fixed my own suggestion which was missing a curly bracket).

Please address the suggestions.

**Comment by jackorp** [comment]
LGTM. Thanks!

**Pull request merged**